### PR TITLE
Profile link fix

### PR DESCRIPTION
**Pull request by ludwiczakpawel**
## What type of PR is this? (check all applicable)

- [x] Bug Fix

## Description
Little coloring fix for hover state for profile name in topbar's dropdown.

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

image

## Added tests?

- [x] no, because they aren't needed

## Added to documentation?

- [x] no documentation needed

### CODE REVIEW

**Comment by rhymes** [comment]
The build fails for an "approvals verify" failure, see https://docs.dev.to/tests/approvals-tests/

**Comment by ludwiczakpawel** [comment]
@rhymes ummm.. sooo i changed a class name (something i've been doing pretty heavily in last several weeks) and now it needs a test?

also, i don't really know anything about test... the code snippet from the docs.. where should I put it?

**Comment by rhymes** [comment]
@ludwiczakpawel good question, I'm not sure how to fix that either.

cc @maestromac

**Comment by rhymes** [comment]
@ludwiczakpawel Ok, I got it:

1. check which file is failing, now `spec/liquid_tags/tweet_tag_spec.rb:25`
1. run `rspec spec/liquid_tags/tweet_tag_spec.rb:25`
1. you'll notice you have a new file on disk, in this case `spec/support/fixtures/approvals/liquid_tweet_tag_spec.received.html`
1. run `approvals verify` and confirm the changes
1. commit the modified file

**Comment by ludwiczakpawel** [comment]
@rhymes done. although I have no idea why since I haven't really touched that file earlier 🤷‍♂

**Comment by maestromac** [comment]
I think it's because there was a change to the twitter asset. That liquid tag also uses it.

**Comment by ludwiczakpawel** [comment]
@maestromac but I haven't touched it...

File: app/assets/javascripts/initializers/initializeBaseUserData.js [hunk 1]
```diff
@@ -104,7 +104,7 @@ function initializeBaseUserData() {
   const user = userData();
   const userProfileLinkHTML =
     '<a href="/' + user.username + '" id="first-nav-link" class="crayons-link crayons-link--block"><div>' +
-    '<span class="fw-medium block color-base-100">' + user.name +'</span>' +
+    '<span class="fw-medium block">' + user.name +'</span>' +
```
[the remark this answers is not in the document]
Comment (ludwiczakpawel) [reply app/assets/javascripts/initializers/initializeBaseUserData.js:107]: yes

**Pull request merged**